echo-terminal.md

```diff
@@ -0,0 +1,31 @@
+# Echo Terminal Snippet Web Component
```

Comment:
Publicly available instructions for using the `nv-echo-terminal` web component.

These snippets have been tested in the documentation.